translations.txt

```diff
@@ -595,63 +595,63 @@ Your Trial key includes all the functionality and device support of a Pro key=De
-If you do not have a backup copy of your license key file you may attempt to recover your key=
-If you want to replace one of your license keys with a new key bound to this USB Flash device please first remove all other key files first=
-Just give us the details about what happened and well do our best to get you up and running again=Just give us the details about what happened and we'll do our best to get you up and running again
-Never self-managehost your flash backups again=
+If this was a Trial installation, you may purchase a license key=Wenn dies eine Test-Installation ist sollten Sie einen Lizenzschlüssel kaufen
+If you do not have a backup copy of your license key file you may attempt to recover your key=Wenn Sie keine Sicherung ihrer Lizenzdatei besitzen sollten Sie versuchen ihren Schlüssel wieder herzustellen
+If you want to replace one of your license keys with a new key bound to this USB Flash device please first remove all other key files first=Wenn Sie eine ihrer Lizenzschlüssel durch einen Neuen, mit diesem USB Flash-Laufwerk verbundenen, ersetzen wollen entfernen Sie bitte alle alten Lizenz Schlüssel Dateien
+Just give us the details about what happened and well do our best to get you up and running again=Bitte geben Sie uns alle verfügbaren Informationen was passiert ist und wir werden unser möglichstes tun Ihnen zu helfen das System wieder zum Laufen zu bringen
+Never self-managehost your flash backups again=Niemals Ihre Flash-Backups selber verwalten
 Not Connected to Mothership=
-Please copy the correct key file to the bootconfig directory on your USB Flash boot device or choose Purchase Key or Replace Key=Please copy the correct key file to the /boot/config directory on your USB Flash boot device or choose Purchase Key or Replace Key
-Please copy the correct key file to the bootconfig directory on your USB Flash boot device or choose Purchase Key=Please copy the correct key file to the /boot/config directory on your USB Flash boot device or choose Purchase Key
-Please try a different USB Flash device=
-Post your diagnostics in the forum for help=
-See also here=
-The key file should be located in the bootconfig directory on your USB Flash boot device=
-This can occur as a result of transferring your license key to a replacement USB Flash device, and you are currently booted from your old USB Flash device=
-Upgrade keys to higher editions=
-While we strive to respond to all requests as quickly as possible please allow for up to 3 business days for a response=
-You have an open pop-up=
-You may purchase a license key corresponding to this USB Flash device to continue using this installation=
-Your flash drive is corrupted or offline=
+Please copy the correct key file to the bootconfig directory on your USB Flash boot device or choose Purchase Key or Replace Key=Bitte kopieren Sie die korrekte Lizenzschlüssel Datei in den /boot/config Ordner auf dem USB Flash Boot-Datenträger oder wählen Sie Schlüssel kaufen oder ersetzen
```

Comment:
"Bitte kopiere die korrekte Lizenzschlüssel Datei in den /boot/config Ordner auf das USB Boot-Laufwerk oder wähle Schlüssel kaufen oder Schlüssel ersetzen"